catalog.bom

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright &copy; 2013 Cloudsoft Corporation Limited
+#
+# Licensed under the Cloudsoft EULA v1.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.cloudsoftcorp.com/cloudsoft-developer-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+brooklyn.catalog:
+  items:
+
+  - id: new-volume-customizer-openstack-vcd
+    version: "0.5.0-SNAPSHOT" # BROOKLYN_BLOCKSTORE_VERSION
+    name: New Volume Customizer OpenStack and vCD
+    description: Attaches additional disks in provisioning time
+    itemType: template
+    license: Apache-2.0
+    publish:
+      license_code: Apache-2.0
+      reference: reference.json
+    item:
+      services:
+      - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
+        id: espentity
+        brooklyn.initializers:
+          - type: io.brooklyn.blockstore.brooklyn-blockstore:brooklyn.location.blockstore.effectors.ExtraHddBodyEffector
```

Comment:
Can we have the catalog item without an effector?
That way it seems they are coupled.
What do you thing about 
- For each cloud Catalog items with initializer.
- For each cloud Catalog items with an effector
- One or (For each cloud) mixed catalog item and effector